bucket/amulet-map-editor.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "0.7.2.8",
+    "description": "Minecraft world editor and converter that supports all versions since Java 1.12 and Bedrock 1.7",
+    "homepage": "https://www.amulet-editor.com",
+    "license": "Unknown",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Amulet-Team/Amulet-Map-Editor/releases/download/0.7.2.8/Amulet-v0.7.2.8-windows-x64.zip",
+            "hash": "b4dcaf42181019b22835b8fd1a50731b0d3fa2475b37f4ce0dc9af1a0fcaf96d",
+            "extract_dir": "Amulet"
+        }
+    },
+    "bin": "amulet_app.exe",
```

Comment:
The app dumps its configuration and logs to the CWD when started from a command prompt, which is bad for usability. It's better to not add it to the `PATH` in this case.

```suggestion
    },
```

Reply:
> Also, should the manifest be named `amulet` or `amulet-editor` instead of `amulet-map-editor`?

I'm not sure. Probably `amulet-editor` or `amulet-map-editor` <https://www.amulet-editor.com/>

Instead of launching from command prompt, would a shortcut be better?

Reply:
![image](https://user-images.githubusercontent.com/51169564/103048856-f7681280-455d-11eb-819d-2d0567a46707.png)

I'm having trouble applying your suggestion, so I'll do it manually in a moment